package.json

```diff
@@ -63,34 +62,32 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "manualBuild": "node verifyCacheVersion.js && yarn localize && d2-manifest package.json ./public/manifest.webapp && REACT_APP_DHIS2_BASE_URL='../../..' react-scripts build && mkdirp buildzip/ && pushd build && bestzip ../buildzip/NewCaptureApp.zip * && popd",
-    "start": "react-scripts start",
-    "build": "node verifyCacheVersion.js && yarn localize && d2-manifest package.json ./public/manifest.webapp && GENERATE_SOURCEMAP=false react-scripts build && cp ./package.json ./build/package.json",
-    "test": "react-scripts test",
+    "start": "yarn verifyCacheVersion && d2-app-scripts start",
+    "start:forCypress": "node scripts/startAppForCypress.js",
+    "build": "yarn verifyCacheVersion && GENERATE_SOURCEMAP=false d2-app-scripts build && cp ./package.json ./build/app/package.json",
+    "build:manual": "yarn verifyCacheVersion && REACT_APP_DHIS2_BASE_URL='../../..' d2-app-scripts build",
+    "test": "d2-app-scripts test",
     "test:debug": "react-scripts --inspect-brk test --runInBand",
-    "eject": "react-scripts eject",
-    "add-flow-types": "flow-typed install",
-    "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
-    "localize": "yarn extract-pot && d2-i18n-generate -n default -p ./i18n/ -o ./src/locales/",
-    "prestart": "node verifyCacheVersion.js && yarn localize && d2-manifest package.json ./public/manifest.webapp",
+    "addFlowTypes": "flow-typed install",
```

Comment:
maybe rename this to `flow:addTypes` to match the `flow:check`

Reply:
I like it! renaming